tasks/sign-index-image/tests/test-sign-index-image-with-pullspec-rewrite.yaml

```diff
@@ -92,32 +92,33 @@ spec:
               #!/usr/bin/env bash
               set -eux
 
-              internalRequest="$(kubectl get internalrequest --sort-by=.metadata.creationTimestamp --no-headers)"
-              params=$(kubectl get internalrequest ${internalRequest} -o jsonpath="{.spec.params}")
+              internalRequest="$(kubectl get internalrequest --sort-by=.metadata.creationTimestamp --no-headers \
+                | awk '$1=$1')"
```

Comment:
I was curious about the `awk` and chatgpt says it's a trick to format the output - instead of multiple spaces, you get just one between fields. But here it's just one field, so it will basically trim the whitespace. Cool. But you could get that directly :)

```suggestion
              internalRequest="$(kubectl get internalrequest --sort-by=.metadata.creationTimestamp --no-headers \
                -o custom-columns=":metadata.name")"
```

I checked and there were no spaces in the output. It's just a test, so up to you if you wanna use this or not.

Also, for various assignment `""` is actually not needed, but I can see it was already there, so fine either way.

Reply:
I like that better than the awk - updated